paramgen_dest.go

```diff
@@ -1,25 +1,181 @@
 // Code generated by paramgen. DO NOT EDIT.
-// Source: github.com/conduitio/conduit-connector-sdk/cmd/paramgen
+// Source: github.com/ConduitIO/conduit-connector-sdk/tree/main/cmd/paramgen
 
-package connectorname
+package rabbitmq
 
 import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
 func (DestinationConfig) Parameters() map[string]sdk.Parameter {
 	return map[string]sdk.Parameter{
-		"destinationConfigParam": {
-			Default:     "yes",
-			Description: "destinationConfigParam must be either yes or no (defaults to yes).",
+		"caCert": {
+			Default:     "",
+			Description: "caCert is the path to the CA certificate to use for TLS",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"clientCert": {
+			Default:     "",
+			Description: "clientCert is the path to the client certificate to use for TLS",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"clientKey": {
+			Default:     "",
+			Description: "clientKey is the path to the client key to use for TLS",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.appID": {
+			Default:     "",
+			Description: "appID specifies the application that created the message.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.contentEncoding": {
+			Default:     "",
+			Description: "contentEncoding specifies the encoding of the message content.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.contentType": {
+			Default:     "text/plain",
+			Description: "contentType specifies the MIME type of the message content. Default is \"text/plain\".",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.correlationID": {
+			Default:     "",
+			Description: "correlationID is used to correlate RPC responses with requests.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.deliveryMode": {
+			Default:     "2",
+			Description: "deliveryMode indicates the message delivery mode. Non-persistent (1) or persistent (2). Default is 2 (persistent).",
```

Comment:
if the valid options are limited, you can add another validation called `inclusion`, so for this param in the config struct, you would add a tag `validation:"inclusion=1|2"`, here are some docs about it https://github.com/ConduitIO/conduit-commons/tree/main/paramgen#:~:text=inclusion%3A%20validates%20that%20the%20parameter%20value%20is%20included%20in%20a%20specified%20list%2C%20this%20list%20values%20are%20separated%20using%20a%20pipe%20character%20%7C.